### ISSUE TITLE
Create My account Page

### ISSUE THREAD

**Issue by mcollinar** [dalenguyen/firebase-wordpress-plugin#63]
I`m trying to create a My account page, but I cannot integrate the logged users with the Users collection.

For example, the user that are trying to log in is hHmTfzqCcgTbMChH2K2dF8AoYES2

and when this user logged in, I want to him see the information that are in this shortcode:
[firestore class='your-class-name' collection_name='Users' document_name='string']
How can I do this?

Describe the solution you'd like
[firestore class='your-class-name' collection_name='Users' document_name='getFirebaseUid' document_uid='id']

When:

document_name='getFirebaseUid' --- the UID that is logged in
document_uid='id' --- the locations of UID in the collections name 'Users'

**Comment by dalenguyen** [comment]
Hi @mcollinar, 

I will be applied like this: 

```
[firestore class='your-class-name' collection_name='Users' document_name='getFirebaseUid' display_fields='some-fields']
```

**Comment by dalenguyen** [comment]
Will be available in v1.1.0
